docker/compose/thehive.app.yaml

```diff
@@ -20,6 +20,7 @@ services:
       - thehive
     networks:
       - vast
+      - thehive
```

Comment:
when `--no-deps` is used, the network needs to be explicitly set (otherwise  `depends_on` is enough to setup the network)